addon/helpers/range.js

```diff
@@ -0,0 +1,37 @@
+import Ember from 'ember';
+import { gte, lte, gt, lt } from '../utils/comparison';
+
+const {
+  Helper: { helper },
+  typeOf
+} = Ember;
+
+export function range([min, max, isInclusive]) {
+  isInclusive = typeOf(isInclusive) === 'boolean' ? isInclusive : false;
+  let testFn = isInclusive ? lte : lt;
+  let numbers = [];
+
+  if (min < max) {
+    for (let i = min; testFn(i, max); i++) {
+      numbers.push(i);
+    }
+  }
+
+  if (min > max) {
```

Comment:
My brain isn't working too well today, I suppose there is probably a better way of doing this

@martndemus